pango/Gir.toml

```diff
@@ -332,6 +331,22 @@ concurrency = "send+sync"
         name = "glyphs"
         const = true
 
+[[object]]
+name = "Pango.Language"
+status = "generate"
+    [[object.function]]
+    pattern = "get_(scripts|preferred)"
+    manual = true
```

Comment:
Add a comment here why. I think this needs an annotation fix, `(array zero-terminated=1)`

Reply:
right, done